Makefile

```diff
@@ -348,20 +348,20 @@ $(GITHOOKSDIR)/%: githooks/%
 	@ln -s ../../$(basename $<) $(dir $@)
 endif
 
-GLOCK := ../../../../bin/glock
-#        ^  ^  ^  ^~ GOPATH
-#        |  |  |~ GOPATH/src
-#        |  |~ GOPATH/src/github.com
-#        |~ GOPATH/src/github.com/cockroachdb
-
 # Update the git hooks and run the bootstrap script whenever any
 # of them (or their dependencies) change.
-.bootstrap: $(GITHOOKS) GLOCKFILE glide.lock
```

Comment:
The downside to this change is that we won't automatically install a dependency when you add a new one. Perhaps it is worthwhile keeping the list of cmds to install in a file but automatically generating that file.